TACC/protx-dashboard
# Hotfix/analytics chart updates

## Overview: ##

Update analytics chart with python code provided by Juliana

**Note** colors are being updated as well for Low, Medium and High

## Related Jira tickets: ##

None

## Summary of Changes: ##
Incorporate changes suggested by Julia in jupyter notebook

## Testing Steps: ##
1.  Go to analytics
2. Select "Predicted Number of Cases"
3. See if you can select a county and show the red line of that county

## Notes: ##

TODO 
- [x] clarify color classes with Joanna/Sophia/Ryan
- [x] clarify issue with annotation colors and histogram bar colors or histogram background color with oanna/Sophia/Ryan
- [x] add line if there is a selected GEOID that there is data for
